plugin/helpers_single_node.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package plugin
+
+import "fmt"
+
+// RunOnSingleNode is a wrapper function which should guarantee that only one plugin instance can run a function associated with the given id at a time
+// The id parameter is an identifier that uses for synchronization must be unique between each function.
+func (p *HelpersImpl) RunOnSingleNode(id string, f func()) (bool, error) {
+	err := p.ensureServerVersion("5.12.0")
+	if err != nil {
+		return false, err
+	}
+
+	id = fmt.Sprintf("runOnSingleNodeLock:%s", id)
```

Comment:
Would you please stay consistent with the rest of the code base and define this in https://github.com/mattermost/mattermost-server/blob/master/plugin/client.go#L10-L13 ?